craco.config.js

```diff
@@ -0,0 +1,25 @@
+const { InjectManifest } = require('workbox-webpack-plugin');
+
+module.exports = {
+  reactScriptsVersion: 'sharetribe-scripts',
+  webpack: {
+    configure: {
+      module: {
+        rules: [
+          {
+            test: /\.m?js$/,
+            resolve: {
+              fullySpecified: false,
+            },
+          },
+        ],
+      },
+    },
+    plugins: [
+      new InjectManifest({
```

Comment:
This is to allow Uppy's Golden Retriever plugin work with large files, see:
https://uppy.io/docs/golden-retriever/#enabling-service-worker